plugins/gateway/kong/plugin.go

```diff
@@ -39,6 +40,12 @@ func New() interface{} {
 	return &Config{}
 }
 
+func (conf Config) Access(kong *pdk.PDK) {
+	if err := kong.Ctx.SetShared(common.RequestTimeContextKey, time.Now().Format(time.RFC3339Nano)); err != nil {
```

Comment:
can't we use `common.GetTimeNowRFC3339Nano()` and set time.Time on the shared context?

Reply:
It is redundant IMO. converting time to string and then to time and then to string again...

Reply:
I just wanted to save `time.Time` as is on the `Shared` - or you can just save string here?

Reply:
Just save string